tronweb/coffee/views.coffee

```diff
@@ -21,9 +21,10 @@ window.dateFromNow = (string, defaultString='never') ->
         """
 
     if string
-        formatted = moment(string).format('MMM, Do YYYY, h:mm:ss a')
+        date = moment.tz string, 'America/Los_Angeles'
```

Comment:
If we are going to hard code this, is it easy to say 

    `(All times in A/L_A time)`

in the html view?